internal/runnerinstall/k8s.go

```diff
@@ -0,0 +1,324 @@
+		"server": map[string]interface{}{
+			"enabled": false,
+		},
+		"runner": map[string]interface{}{
+			"id": opts.Id,
+			"image": map[string]interface{}{
+				"repository": i.Config.RunnerImage,
+				"tag":        i.Config.RunnerImageTag,
+			},
+			"resources": map[string]interface{}{
+				"requests": map[string]interface{}{
+					"memory": i.Config.MemRequest,
+					"cpu":    i.Config.CpuRequest,
+				},
+			},
+			"server": map[string]interface{}{
+				"addr":          opts.ServerAddr,
+				"tls":           opts.AdvertiseClient.Tls,
+				"tlsSkipVerify": opts.AdvertiseClient.TlsSkipVerify,
+				"cookie":        opts.Cookie,
+			},
+			"serviceAccount": map[string]interface{}{
+				"create": i.Config.CreateServiceAccount,
+			},
+
+			"pullPolicy": "always",
+		},
+	}
+	s = sg.Add("Installing Waypoint Helm chart with runner options: " + c.Name())
+	_, err = client.RunWithContext(ctx, c, values)
+	if err != nil {
+		return err
+	}
+	s.Update("Waypoint runner installed with Helm!")
+	s.Status(terminal.StatusOK)
+	s.Done()
+
+	return nil
+}
+
+func (i *K8sRunnerInstaller) InstallFlags(set *flag.Set) {
+	set.StringVar(&flag.StringVar{
+		Name:   "k8s-Config-path",
+		Usage:  "Path to the kubeconfig file to use,",
+		Target: &i.Config.KubeconfigPath,
+	})
+
+	set.StringVar(&flag.StringVar{
+		Name:   "k8s-context",
+		Target: &i.Config.K8sContext,
+		Usage: "The Kubernetes context to install the Waypoint runner to. If left" +
+			" unset, Waypoint will use the current Kubernetes context.",
+	})
+
+	set.StringVar(&flag.StringVar{
+		Name:   "k8s-helm-version",
+		Target: &i.Config.Version,
+		Usage:  "The version of the Helm chart to use for the Waypoint runner install.",
+	})
+
+	set.StringVar(&flag.StringVar{
+		Name:    "k8s-namespace",
+		Target:  &i.Config.Namespace,
+		Default: "default",
+		Usage: "The namespace in the Kubernetes cluster into which the Waypoint " +
+			"runner will be installed.",
+	})
+
+	set.StringVar(&flag.StringVar{
+		Name:    "k8s-runner-image",
+		Target:  &i.Config.RunnerImage,
+		Default: defaultRunnerImage,
+		Usage:   "Docker image for the Waypoint runner.",
+	})
+
+	set.StringVar(&flag.StringVar{
+		Name:    "k8s-runner-image-tag",
+		Target:  &i.Config.RunnerImageTag,
+		Default: defaultRunnerImageTag,
+		Usage:   "Tag of the Docker image for the Waypoint runner.",
+	})
+
+	set.StringVar(&flag.StringVar{
+		Name:    "k8s-cpu-request",
+		Target:  &i.Config.CpuRequest,
+		Default: defaultRunnerCPU,
+		Usage:   "Requested amount of CPU for Waypoint runner.",
+	})
+
+	set.StringVar(&flag.StringVar{
+		Name:    "k8s-mem-request",
+		Target:  &i.Config.MemRequest,
+		Default: defaultRunnerMemory,
+		Usage:   "Requested amount of memory for Waypoint runner.",
+	})
+
+	set.BoolVar(&flag.BoolVar{
+		Name:    "k8s-runner-service-account-init",
+		Target:  &i.Config.CreateServiceAccount,
+		Default: true,
+		Usage:   "Create the service account if it does not exist. The default is true.",
```

Comment:
You actually don't need this! Our flag package automatically informs the user what the `Default` is if that field is set.
```suggestion
		Usage:   "Create the service account if it does not exist.",
```
For example, right now this is returning:
```
 -k8s-runner-service-account-init
      Create the service account if it does not exist. The default is true.
      The default is true.
```

Reply:
Oooh fascinating though, because our docs generation does not include what is generated for the `Default` field, just our cli `-h` does. I still say remove this here, and including the `Default` infos will be part of the revamp of how we generate documentation.

Reply:
Good call out, I went ahead and removed the snippet in `4dc7f1ef6`.